dts/arm/st/h7/stm32h7.dtsi

```diff
@@ -546,6 +546,26 @@
 				#pwm-cells = <2>;
 			};
 		};
+
+		adc12: adc@40022000 {
```

Comment:
I know this is used in STM32 packages, but I wonder if adc1_2 would best for disambiguation.